client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -165,6 +178,19 @@ private void initializeLifecycleManager(String appId) {
             lifecycleManager.registerInvalidatedBroadcastCallback(
                 shuffleId -> SparkUtils.invalidateSerializedGetReducerFileGroupResponse(shuffleId));
           }
+          if (lifecycleManager.conf().clientFetchCleanFailedShuffle()) {
```

Comment:
These lines are duplicates of lines 159~172.

Reply:
oops, some merge error, fixed